jrnl/jrnl.py

```diff
@@ -323,10 +323,31 @@ def _display_search_results(args, journal, **kwargs):
         print(plugins.get_exporter("tags").export(journal))
 
     elif args.export:
-        exporter = plugins.get_exporter(args.export)
-        print(exporter.export(journal, args.filename))
+        _export_journal(args, journal)
```

Comment:
This function feels awkward here. It's extending the conditional block, but now the conditions are in two places. Do you mind merging in the conditions into this same block?

It might look something like this:
```python
...
    if args.export == "pretty":
        print(journal.pprint(short=False))

    elif args.export == "short":
        print(journal.pprint(short=True))

    elif args.export:
        _export_journal(args, journal)
...
```